### PR TITLE
applying changes from internal

### PR DESCRIPTION
**Pull request by bartdevylder**
These are changes done on internal not present here yet.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -59,12 +65,13 @@ function run {
...
         -e "ENABLE_HTTPS=$ENABLE_HTTPS" \
         -e "ANOMALY_EMAIL=$ANOMALY_EMAIL" \
+        -e "COSCALE_VERSION=$IMAGE_VERSION" \
         --name coscale_$SERVICE coscale/$SERVICE:$IMAGE_VERSION
 }
 
 # Run the data services
 for SERVICE in $DATA_SERVICES; do
-    if [ "$NAME" == "all" ] || [ "$NAME" == "data" ] || [ "$NAME" == "$SERVICE" ]; then
+    if [ "$NAME" == "all" ] || [ "$NAME" == "$SERVICE" ]; then
```
Comment (fryckbos) [inline run.sh]: I would like to keep the "data" option, sometimes it is handy to start the data services first without the CoScale services.

File: stop.sh [hunk 2]
```diff
@@ -16,13 +16,13 @@ function stop {
...
-    docker stop coscale_$SERVICE
-    docker rm coscale_$SERVICE
+    docker stop coscale_$SERVICE || echo "(Container not running)"
+    docker rm coscale_$SERVICE || echo "(Container not present)"
 }
 
 # Stop the data services
 for SERVICE in $DATA_SERVICES; do
-    if [ "$NAME" == "all" ] || [ "$NAME" == "data" ] || [ "$NAME" == "$SERVICE" ]; then
+    if [ "$NAME" == "all" ] || [ "$NAME" == "$SERVICE" ]; then
```
Comment (fryckbos) [inline stop.sh]: Same thing here.

**Review by fryckbos — CHANGES_REQUESTED** [review]
2 small fixes.

File: run.sh (hunk 1 above)
Comment (bartdevylder) [reply run.sh]: yep. Somehow this did not exist on internal yet so was mistakenly removed

**Review by fryckbos — APPROVED** [review]

**Pull request merged**